react/components/FixedContent.js

```diff
@@ -0,0 +1,120 @@
+import React, { PureComponent, Fragment } from 'react'
+import { bool, string } from 'prop-types'
+import { ExtensionPoint, withRuntimeContext } from 'vtex.render-runtime'
+import Logo from './Logo'
+import SearchBar from './SearchBar'
+import Icons from './Icons'
+import { CONSTANTS } from './Helpers'
+import { Container } from 'vtex.store-components'
+import header from '../store-header.css'
+
+/**
+ * Component that deals with content thats always fixed on top.
+ * Also handles the toggle between mobile search and mobile navbar.
+ * TODO: Change to hooks when it's available at render.
+ */
+class FixedContent extends PureComponent {
+
+  static propTypes = {
```

Comment:
Choose one style of adding propTypes... I thinks the majority of our components declare it outside the class closure and after its declaration.

Reply:
✅